models/stg_recharge.yml

```diff
@@ -587,5 +587,3 @@ models:
         description: Additional comment for cancellation.
       - name: is_most_recent_record
         description: Boolean indicating whether record was the most recent instance.
-      - name: _fivetran_synced
```

Comment:
the column is still included in the model right? So should we keep the docs for it?

Reply:
@fivetran-jamie I meant to remove it completely since it was replaced by `updated_at` in the subscription history 
`row_number() over (partition by subscription_id order by updated_at desc) = 1 as is_most_recent_record`
but I see it's still in the source macro. I'm not sure if it's useful anymore, and it's not included in the other models, so I'm thinking I'll remove from the macro too? Or what do you think about `_fivetran_synced`?

Reply:
Added back in per our discussion earlier!